libs/template/spec/let/let.directive.strategy.spec.ts

```diff
@@ -1,28 +1,30 @@
 import { ChangeDetectorRef, Component, TemplateRef, ViewContainerRef } from '@angular/core';
+import { RenderStrategy } from '@rx-angular/template';
```

Comment:
This is an interesting one! 
I have no clue how we should test if the typing really works... (in the template) :D

Reply:
@hoebbelsB Yeah, I could do that :) Could you just explain why should I import it with a relative path rather than what is right now?

Reply:
@BioPhoton I have no clue how could we could test the `ngTemplateContextGuard`. 
1. It seems still no working properly with Ivy.
2. I was curious and checked the source code for the `ngIf` directive's test suite inside the Angular project and found no test concerning `ngTemplateContextGuard`.
Conclusion: I don't think we should think too much about it (at least now).

Reply:
Agreed. Let's at least test that it is implemented and we are good to go.

Reply:
@kajetansw I thought it was a good idea to limit the possibilities for having cyclic dependencies because of some .spec file. Also we are "living" this practice  in any other specs if I remember correctly. So just to be consistent. But other than that, there is no real reason behind it I guess. So feel free to leave it as is :+1:

Reply:
@hoebbelsB Like you can see, I changed it.